mistralrs-bench/src/main.rs

```diff
@@ -313,14 +313,7 @@ fn main() -> anyhow::Result<()> {
     if use_flash_attn {
         info!("Using flash attention.");
     }
-    if use_flash_attn
-        && matches!(
-            loader.get_kind(),
-            ModelKind::QuantizedGGML
-                | ModelKind::QuantizedGGUF
-                | ModelKind::XLoraGGML
-                | ModelKind::XLoraGGUF
-        )
+    if use_flash_attn && !loader.get_kind().is_quantized()
```

Comment:
This is now simpler and covers both adapter kinds as a bugfix.

**NOTE:** `Speculative` kind will also trigger this now. I am not 100% sure if that's desired? This is due to the `any()` call, as I assume if either `target` or `draft` are quantized it may not be compatible, but likewise this does not guarantee `is_quantized == true` equates to `Speculative` having both as true.